locale/zh/messages.json

```diff
@@ -329,7 +331,7 @@
   "Error finding node": "查找节点错误",
   "Error finding {item}": "查找 {item} 错误",
   "Exceeded licensed node count on one occasion.": "超过许可节点数 1 次。",
-  "Exceeded licensed node count on {numberBreaches} occasions.": "超过许可节点数 {numberBreaches} 次。",
```

Comment:
Why did this get replaced? The message still exists, but the translation was removed.

Reply:
🤔 I don't know. I'll see about checking out the messages from master and doing another extract/compile

Reply:
@brandonc that fixed it.